libraries/SRV_Channel/SRV_Channel.h

```diff
@@ -376,7 +376,7 @@ class SRV_Channels {
     static void set_output_pwm_chan_timeout(uint8_t chan, uint16_t value, uint16_t timeout_ms);
 
     // set output value for a function channel as a scaled value. This
-    // calls calc_pwm() to also set the pwm value
```

Comment:
Maybe the comment should be, "this should be **followed** by a call to calc_pwm() to output the pwm values"

Reply:
done